src/output.ts

```diff
@@ -187,7 +187,9 @@ export default class Output {
 
 				let offsetY = 0;
 
-				for (let line of lines) {
+				for (let index = 0; index < lines.length; index++) {
```

Comment:
```suggestion
				for (let [index, line] of lines.entries()) {
```

Reply:
Ah, yeah, that works, too, I guess.  I'll use that.